jerry-core/parser/js/js-scanner-internal.h

```diff
@@ -16,7 +16,7 @@
 #ifndef JS_SCANNER_INTERNAL_H
 #define JS_SCANNER_INTERNAL_H
 
-/** \addtogroup parser Parser
+/* \addtogroup parser Parser
```

Comment:
Is this change really needed here?

Reply:
Somehow I get doxygen error messages without it.